### PR TITLE
Change old scheme to new one and Add simple sample.

### PR DESCRIPTION
**Pull request by NT-D**
Because 2 reasons, I modified the document. Would you review and merge this pull request?

- As described with [this document](https://msdn.microsoft.com/en-us/microsoft-teams/schemamigrate), `needsIdentity` was ver0.4 scheme and need to change ver1.0 one. Therefore I changed `needsIdentity` to `permissions` array.
- Because I thought simple sample will help users to understand well, I added it in this document.

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/getusercontext.md](https://github.com/NT-D/Microsoft-teams-docs/blob/masota/ModifyGetusercontext/teams/getusercontext.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C21%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201705211224533649-156%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: teams/getusercontext.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Context about the user, team or company can be especially useful when:
 
 >**Important:** While this user information can help provide a smooth user experience, you should **not** use it as proof of identity. For example, an attacker could you load your page in a 'bad browser' and provide it with any information they want.
 
-To obtain this identifying information about the user, team or company, you must set the `needsIdentity` element in your tab manifest to `true`. Doing so will prompt the user for their consent when they add your tab.  
+To obtain this identifying information about the user, team or company, you must add the `permissions` element (it is array) in your tab manifest and add `identity` in the array as described in [this document](https://msdn.microsoft.com/en-us/microsoft-teams/schema#permissions). Doing so will prompt the user for their consent when they add your tab.
```
Comment (dfederm) [inline teams/getusercontext.md]: Please use a relative link to schema.md#perissions

Comment (SheriEllisMSFT) [inline teams/getusercontext.md]: I'd like to clarify the wording a bit:

To obtain this identifying information about the user, team or company, you must add the identity element to the permissions object in your manifest in an array as described in [this document](schema.md#permissions). Doing so will prompt the user for their consent when they add your tab.

Comment (SheriEllisMSFT) [inline teams/getusercontext.md]: About the sample, I'd like to ask @dfederm to review it. Also, I'd like to put it before "In addition...". Also, can we make "In addition..." its own header? Guessing I need to submit a different PR? Sorry for the newbie questions. Thanks!

**Comment by RichMoeMSFT** [comment]
Changed added directly to broader refactor
